mscore/pagesettings.cpp

```diff
@@ -97,10 +97,10 @@ void PageSettings::hideEvent(QHideEvent* ev)
 //   setScore
 //---------------------------------------------------------
 
-void PageSettings::setScore(MasterScore* s)
+void PageSettings::setScore(Score* s)
       {
-      cs  = s;
-      MasterScore* sl = s->clone();
+	  cs = static_cast<MasterScore*>(s);
```

Comment:
whitespace damage, we indent by 6 spaces not tabs

Reply:
Noted. First time for me.

Will get to work in these two